ingest.go

```diff
@@ -732,20 +847,30 @@ func (d *DB) ingest(
 		for i := len(d.mu.mem.queue) - 1; i >= 0; i-- {
 			m := d.mu.mem.queue[i]
 			if ingestMemtableOverlaps(d.cmp, m, meta) {
-				mem = m
-				if mem.flushable == d.mu.mem.mutable {
-					err = d.makeRoomForWrite(nil)
+				if (len(d.mu.mem.queue) > d.opts.MemTableStopWritesThreshold-1) ||
```

Comment:
Why is " len(d.mu.mem.queue) > d.opts.MemTableStopWritesThreshold-1 " needed here, if we only flush one mem and then proceed to ingest as ingestflushable?

Reply:
When we ingest-as-flushable, we still need to create a new memtable for subsequent writes. Construction of this new memtable can put us over the limit set by `MemTableStopWritesThreshold`.

Looking at this now, I'm not sure why it's not size-aware. Eg,:
```
			var size uint64
			for i := range d.mu.mem.queue {
				size += d.mu.mem.queue[i].totalBytes()
			}
			if size >= uint64(d.opts.MemTableStopWritesThreshold)*uint64(d.opts.MemTableSize) ||
			// ...
```

Reply:
Thanks for you reply. Do we have test cases to illustrate the benefits of ingest-as-flushable？